source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRColumnRenderer.h

```diff
@@ -9,6 +9,11 @@
 
 @interface ACRColumnRenderer:ACRBaseCardElementRenderer
 
+@property BOOL containerHasFillAlignment;
+
 + (ACRColumnRenderer *)getInstance;
 
+- (void)setFillAlignment;
```

Comment:
you don't need getter and setter for property.
you can access it by '.' operator. example.fillAlignment = YES; example.fillAlignment = NO;